src/dvc_render/vega_templates.py

```diff
@@ -620,64 +593,28 @@ class LinearTemplate(Template):
         "title": Template.anchor("title"),
         "width": 300,
         "height": 300,
-        "layer": [
-            {
-                "encoding": {
-                    "x": {
-                        "field": Template.anchor("x"),
-                        "type": "quantitative",
-                        "title": Template.anchor("x_label"),
-                    },
-                    "y": {
-                        "field": Template.anchor("y"),
-                        "type": "quantitative",
-                        "title": Template.anchor("y_label"),
-                        "scale": {"zero": False},
-                    },
-                    "color": {"field": "rev", "type": "nominal"},
-                    "tooltip": [
-                        {"field": "<DVC_METRIC_X>", "type": "quantitative"},
-                        {"field": "<DVC_METRIC_Y>", "type": "quantitative"},
-                        {"field": "rev", "type": "nominal"},
-                    ],
-                },
-                "layer": [
-                    {"mark": "line"},
-                    {
-                        "transform": [
-                            {"filter": {"param": "hover", "empty": False}}
-                        ],
-                        "mark": "circle",
-                    },
-                    {
-                        "mark": "rule",
-                        "params": [
-                            {
-                                "name": "hover",
-                                "select": {
-                                    "type": "point",
-                                    "on": "mouseover",
-                                    "nearest": True,
-                                },
-                            }
-                        ],
-                        "encoding": {
-                            "color": {
-                                "condition": {
-                                    "param": "hover",
-                                    "empty": False,
-                                    "value": "1",
-                                },
-                                "value": "transparent",
-                            }
-                        },
-                    },
-                ],
-            }
-        ],
+        "mark": {"type": "line", "point": True, "tooltip": True},
+        "encoding": {
+            "x": {
+                "field": Template.anchor("x"),
+                "type": "quantitative",
+                "title": Template.anchor("x_label"),
+            },
+            "y": {
+                "field": Template.anchor("y"),
+                "type": "quantitative",
+                "title": Template.anchor("y_label"),
+                "scale": {"zero": False},
+            },
+            "color": {"field": "rev", "type": "nominal"},
+        },
     }
 
 
+class SimpleLinearTemplate(LinearTemplate):
+    DEFAULT_NAME = "simple"
```

Comment:
I think it no longer makes sense to have a `simple` linear template. Keeping this for backwards compatibility